parametric/test_library_tracestats.py

```diff
@@ -151,8 +151,7 @@ def test_distinct_aggregationkeys_TS003(library_env, test_agent, test_library, t
             span.set_meta(key="http.status_code", val="400")
 
     if test_server.lang == "golang":
```

Comment:
can we remove this check completely? `flush` should be called by the test fixture

Reply:
Ah yes, just removed the checks!